packages/indexer-agent/src/db/migrations/16-modify-invalid-receipts-table.ts

```diff
@@ -0,0 +1,51 @@
+import { Logger } from '@graphprotocol/common-ts'
+import { QueryInterface, DataTypes } from 'sequelize'
+
+interface MigrationContext {
+  queryInterface: QueryInterface
+  logger: Logger
+}
+
+interface Context {
+  context: MigrationContext
+}
+
+export async function up({ context }: Context): Promise<void> {
+  const { queryInterface, logger } = context
+
+  const tables = await queryInterface.showAllTables()
+  logger.debug(
+    `Modifying tables scalar_tap_receipts_invalid to add extra column`,
+  )
+
+  if (tables.includes('scalar_tap_receipts_invalid')) {
+    await queryInterface.addColumn('scalar_tap_receipts_invalid', 'error_log', {
+      type: DataTypes.TEXT,
+      allowNull: false,
+    })
+  }
```

Comment:
Could you add a "default value" for the current rows? Else, this migration will break in case an indexer contains any invalid receipt.

Also, could you add a default value for new inserts so this is a non-breaking change?